From b5d4a4997a68ba38679168dc045054c90bba2099 Mon Sep 17 00:00:00 2001
From: Raffaele Aquilone <raquiloX>
Date: Tue, 5 Mar 2013 11:33:55 +0100
Subject: [PORT FROM MAIN][BT] Fix read buffer size for PAN on TI chip

BZ: 83501

Fix the read buffer size used by Bluedroid to read packets that
exceed the expected MTU coming from TI chip, mandatory for PAN

Change-Id: Id4a7cca4d1efb169b015ab9a9ca45ff5682fc4fb
Orig-Change-Id: I22b6e95781d29bdbdaeafe6cf3fe2087a0d16aef
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/95089
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
---
 hci/include/bt_hci_bdroid.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/hci/include/bt_hci_bdroid.h b/hci/include/bt_hci_bdroid.h
index 7b23851..97779e1 100644
--- a/hci/include/bt_hci_bdroid.h
+++ b/hci/include/bt_hci_bdroid.h
@@ -63,7 +63,8 @@
 #endif  // (BTHC_LINUX_BASE_POLICY != SCHED_NORMAL)
 
 #ifndef BTHC_USERIAL_READ_MEM_SIZE
-#define BTHC_USERIAL_READ_MEM_SIZE (1024)
+/* Buffer size value fixed for supporting any possible chip */
+#define BTHC_USERIAL_READ_MEM_SIZE (1026 + BT_HC_HDR_SIZE)
 #endif
 
 #ifndef BTSNOOPDISP_INCLUDED
-- 
1.7.4.1

